packages/components/src/PassengerCard/PassengerCard.js

```diff
@@ -0,0 +1,135 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text, StyleSheet, Icon } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import BagInformation from './BagInformation';
+import Separator from '../separator/Separator';
+
+type Bag = {|
+  +count: number,
+  +type: string,
+|};
+
+type Props = {|
+  +blank: boolean,
```

Comment:
We can make all (just `passengerCount` should be defined) props optional with `''` value (if necessary) as `defaultProps`. And then `blank` prop will not be necessary. 
Then we will need just one condition below for `name` where `X. Passenger` will be used in case of empty name.

Reply:
It would be better than with `blank` prop. Thanks!